test.cpp

```diff
@@ -40,7 +40,7 @@ void test()
         }
     )";
 
-    slang::wgsl::Module* module1 = session->loadModuleFromSource(source1.c_str());
+    slang::wgsl::Module* module1 = session->loadModuleFromSource(source1.c_str(), "user", "/user.slang");
```

Comment:
We need to make the build.mk working again to build the local test.

Reply:
I think the right way is to test these APIs in javascript with using some node.js unit test frameworks.

Reply:
I used cpp at first place is because it's easier to debug if we have any bugs in the wasm wrapper code.

Reply:
That's fair, but then this test should probably live in the slang repo?

Reply:
Right.